test/online/fetch_test.rb

```diff
@@ -54,7 +54,13 @@ def test_fetch_over_https_with_certificate_callback_fail
         })
       end
 
-      assert_equal "user rejected certificate for github.com", exception.message
+      response_message = if Gem.win_platform?
```

Comment:
the windows response is actually more correct to the scenario. strange how we get different messages for the same action.